string	type
int	task_id
int	instance_id
timeval	time
int	agent_id
int	checksum
